src/integration-tests/src/sql.rs

```diff
@@ -74,12 +76,10 @@ pub async fn run_sql_instances_service(
         .list()
         .set_project(&project_id)
         .set_filter(format!("name:{name}"))
-        .send()
-        .await?;
-    println!("SUCCESS on list sql instance: {list:?}");
-    // TODO(#2067) - these assertions are flaky, disabled for now
-    // assert_eq!(list.items.len(), 1);
-    // assert!(list.items.into_iter().any(|v| v.name.eq(&name)));
+        .by_item()
+        .into_stream();
+    println!("SUCCESS on list sql instance");
+    assert_eq!(list.count().await, 1);
```

Comment:
I think this reintroduces the flake I fixed in #2117, consider a different assertion that does not depend on getting exactly one response (sometimes we seem to get 0).